### PR TITLE
Bilibili embed iframe

### PR DESCRIPTION
**Pull request by LavaCxx**
---

- [x] I understand that contributing to this repository will require me to agree with the [CLA](https://github.com/anyproto/.github/blob/main/docs/CLA.md)

---

### Description
I forgot to support the original iframe string.

### What type of PR is this? (check all applicable)

- [x] 🍕 Feature
- [ ] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI

### Mobile & Desktop Screenshots/Recordings

image

### Added tests?

- [ ] 👍 yes
- [x] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

### Added to documentation?

- [ ] 📜 README.md
- [ ] 📓 [tech-docs](https://github.com/anyproto/tech-docs)
- [x] 🙅 no documentation needed

### CODE REVIEW

File: src/ts/lib/util/embed.ts [hunk 1]
```diff
@@ -168,6 +168,12 @@ class UtilEmbed {
 			};
 
 			case I.EmbedProcessor.Bilibili: {
+				const iframeUrl = url.match(/src="([^"]+)"/);
+				if (iframeUrl) {
+					url = iframeUrl[1];
+					return;
```
Comment (ra3orblade) [inline src/ts/lib/util/embed.ts:174]: return is incorrect here, break

**Review by ra3orblade — CHANGES_REQUESTED** [review]

**Comment by ra3orblade** [comment]
Could you please explain what are you doing here? Embed url only checks url, not the source html code.

**Comment by LavaCxx** [comment]
> Could you please explain what are you doing here? Embed url only checks url, not the source html code.

Sorry, I have already re-reviewed the code and modified the logic.

**Comment by ra3orblade** [comment]
> > Could you please explain what are you doing here? Embed url only checks url, not the source html code.
> 
> Sorry, I have already re-reviewed the code and modified the logic.

Update from main pls, there is already logic for Bilibili to parse schemeless urls `// Fix Bilibili schemeless urls
					if (block.isEmbedBilibili() && text.match(/src="\/\/player[^"]+"/)) {
						text = text.replace(/src="(\/\/player[^"]+)"/, 'src="https:$1"');
					};`

Could you please combine these two logics under one `if` statement?

**Comment by ra3orblade** [comment]
Something is wrong with your branch.

**Comment by LavaCxx** [comment]
> Something is wrong with your branch, conflicts and shows changes that are already in main.

I'm not very good at solving git issues, maybe I should create a new pr.

**Pull request closed without merging**